jwst/extract_1d/extract.py

```diff
@@ -197,39 +197,41 @@ def get_extract_parameters(ref_dict,
         assigned if `ref_dict` is None.  For a reference image, the key
         'ref_image' gives the (open) image model.
     """
-
-    extract_params = {'match': NO_MATCH}        # initial value
+    extract_params = {'match': NO_MATCH}  # initial value
 
     if ref_dict is None:
         # There is no reference file; use "reasonable" default values.
         extract_params['ref_file_type'] = FILE_TYPE_OTHER
         extract_params['match'] = EXACT
         shape = input_model.data.shape
         extract_params['spectral_order'] = sp_order
-        extract_params['xstart'] = 0                    # first pixel in X
-        extract_params['xstop'] = shape[-1] - 1         # last pixel in X
-        extract_params['ystart'] = 0                    # first pixel in Y
-        extract_params['ystop'] = shape[-2] - 1         # last pixel in Y
+        extract_params['xstart'] = 0  # first pixel in X
+        extract_params['xstop'] = shape[-1] - 1  # last pixel in X
+        extract_params['ystart'] = 0  # first pixel in Y
+        extract_params['ystop'] = shape[-2] - 1  # last pixel in Y
         extract_params['extract_width'] = None
         extract_params['src_coeff'] = None
         extract_params['bkg_coeff'] = None
+
         if apply_nod_offset is None:
             extract_params['apply_nod_offset'] = True
         else:
             extract_params['apply_nod_offset'] = apply_nod_offset
+
         extract_params['nod_correction'] = 0
         extract_params['independent_var'] = 'pixel'
         extract_params['smoothing_length'] = 0  # because no background sub.
-        extract_params['bkg_order'] = 0         # because no background sub.
-        # Note that extract_params['dispaxis'] is not assigned.  This will
-        # be done later, possibly slit by slit.
-
+        extract_params['bkg_order'] = 0  # because no background sub.
+        # Note that extract_params['dispaxis'] is not assigned.  This will be done later, possibly slit by slit.
     elif ref_dict['ref_file_type'] == FILE_TYPE_JSON:
         extract_params['ref_file_type'] = ref_dict['ref_file_type']
+
         for aper in ref_dict['apertures']:
-            if 'id' in aper and aper['id'] != "dummy" and \
-               (aper['id'] == slitname or aper['id'] == "ANY" or
-                slitname == "ANY"):
+            if (
+                    'id' in aper
+                    and aper['id'] != "dummy"
+                    and (aper['id'] == slitname or aper['id'] == "ANY" or slitname == "ANY")
+            ):
```

Comment:
Just curious - was this block produced by an automatic code formatter?

Reply:
Nope! 
It's definitely a style that's similar to some common code formatters though (I think [Black](https://pypi.org/project/black/) does something similar), but for most of this I was doing it "by hand" with the help of an IDE (which does help with automatically adjusting indentation and such to match some standard) so that I could also work through the logic as I went